libs/remix-tests/src/compiler.ts

```diff
@@ -172,14 +172,7 @@ export function compileFileOrFiles (filename: string, isDirectory: boolean, opts
  */
 export function compileContractSources (sources: SrcIfc, compilerConfig: CompilerConfiguration, importFileCb: any, opts: any, cb): void {
   let compiler
-  const accounts: string[] = opts.accounts || []
   const filepath = opts.testFilePath || ''
```

Comment:
you don't need anymore to inject the sources?

Reply:
I think no, because they are received with content-import dynamically so we don't need to add them dynamically